tensorflow/contrib/signal/python/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Well, this file by @martinwicke isn't either:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/nn/python/__init__.py
Is there any special intent behind this?